### PR TITLE
feat: stripe connect atom

### PR DESCRIPTION
**Pull request by Ryukemeister**
## What does this PR do?

- Adds an atom for a user to connect to stripe `StripeConnect`, we need this for event types atom

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [ ] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com). If N/A, write N/A here and check the checkbox.
- [ ] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?
This can be tested in the examples app under `packages/platform/example/base`

### CODE REVIEW

File: apps/api/v2/src/modules/stripe/stripe.service.ts [hunk 1]
```diff
@@ -1,15 +1,130 @@
...
+
+    const { client_id, client_secret } = stripeKeysResponseSchema.parse(app?.keys);
+
+    if (!client_id) {
+      throw new NotFoundException();
+    }
+
+    if (!client_secret) {
+      throw new NotFoundException();
+    }
```
Comment (ThyMinimalDev) [inline apps/api/v2/src/modules/stripe/stripe.service.ts:72]: let's add a string message within the exception "NotFoundException('Stripe app not found');

File: apps/api/v2/src/modules/stripe/controllers/stripe.controller.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    @Req() req: Request,
+    @Headers("Authorization") authorization: string,
+    @GetUser() user: UserWithProfile,
+    @Query("redir") redir?: string | null
+  ): Promise<StripConnectOutputResponseDto> {
+    const origin = req.headers.origin;
+    const accessToken = authorization.replace("Bearer ", "");
+
+    const state = {
+      onErrorReturnTo: origin,
```
[the remark this answers is not in the document]
Comment (Ryukemeister) [reply apps/api/v2/src/modules/stripe/controllers/stripe.controller.ts:53]: Think I'll just add it in this PR itself.

**Review by ThyMinimalDev — DISMISSED** [review]
LGTM

**Comment by Ryukemeister** [comment]
Will fix the merge conflicts now.

**Review by ThyMinimalDev — APPROVED** [review]

**Pull request merged**